### ISSUE TITLE
Admin settings general template error

### ISSUE THREAD

**Issue by rauwebieten** [craftcms/cms#7350]
### Description

After install, when visiting /admin/settings/general
There is a twig runtime exception: Key "name" for array with keys "edition, schemaVersion" does not exist.

https://snipboard.io/Kr9e1E.jpg

```
Twig\Error\RuntimeError: Key "name" for array with keys "edition, schemaVersion" does not exist. in /vagrant/vendor/craftcms/cms/src/templates/settings/general/_index.html:30
Stack trace:
#0 /vagrant/vendor/craftcms/cms/src/helpers/Template.php(106): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Array, 'name', Array, 'any', false, false)
#1 /vagrant/storage/runtime/compiled_templates/fe/fec9c85d75ca82522fbbb560b38db17dd4feb3bd5f30d0ca1c29959d015acdd5.php(75): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Array, 'name', Array)
#2 /vagrant/vendor/twig/twig/src/Template.php(184): __TwigTemplate_401533d7b530d62a4d9951e588c966c5371dc5e7918c25cc776fb24cc9bbe0dd->block_content(Array, Array)
#3 /vagrant/storage/runtime/compiled_templates/ac/ac9ae7f43a8bf0d30db82a15743ef2a4c86b868a6ebd279d7d47041bb8b3a7ff.php(495): Twig\Template->displayBlock('content', Array, Array)
#4 /vagrant/vendor/twig/twig/src/Template.php(184): __TwigTemplate_285324850b71b0c462fb4abca90e0a2b3cd4dc9a14da740fdaeaa2bfe5f2e518->block_main(Array, Array)
#5 /vagrant/storage/runtime/compiled_templates/ac/ac9ae7f43a8bf0d30db82a15743ef2a4c86b868a6ebd279d7d47041bb8b3a7ff.php(314): Twig\Template->displayBlock('main', Array, Array)
#6 /vagrant/vendor/twig/twig/src/Template.php(184): __TwigTemplate_285324850b71b0c462fb4abca90e0a2b3cd4dc9a14da740fdaeaa2bfe5f2e518->block_body(Array, Array)
#7 /vagrant/storage/runtime/compiled_templates/e3/e35ca60ac39f9a1cf29c0d185d12b91d7ad6ac5b0ceb2228467066fc349541e7.php(85): Twig\Template->displayBlock('body', Array, Array)
#8 /vagrant/vendor/twig/twig/src/Template.php(407): __TwigTemplate_18d9602f11be87a4c29a2e4db5ebc8dcbfb631bcd46f3d74988e582b40b21ebf->doDisplay(Array, Array)
#9 /vagrant/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#10 /vagrant/storage/runtime/compiled_templates/3c/3c43a38253baa3fd5e5b05ef32499be7e3773913825b2cb9a12a558a78ec2266.php(56): Twig\Template->display(Array, Array)
#11 /vagrant/vendor/twig/twig/src/Template.php(407): __TwigTemplate_5121b96418005028cf272de8d7dfdf3de2cd6fa9ca956311d885fddc15379750->doDisplay(Array, Array)
#12 /vagrant/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#13 /vagrant/storage/runtime/compiled_templates/ac/ac9ae7f43a8bf0d30db82a15743ef2a4c86b868a6ebd279d7d47041bb8b3a7ff.php(156): Twig\Template->display(Array, Array)
#14 /vagrant/vendor/twig/twig/src/Template.php(407): __TwigTemplate_285324850b71b0c462fb4abca90e0a2b3cd4dc9a14da740fdaeaa2bfe5f2e518->doDisplay(Array, Array)
#15 /vagrant/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#16 /vagrant/storage/runtime/compiled_templates/fe/fec9c85d75ca82522fbbb560b38db17dd4feb3bd5f30d0ca1c29959d015acdd5.php(54): Twig\Template->display(Array, Array)
#17 /vagrant/vendor/twig/twig/src/Template.php(407): __TwigTemplate_401533d7b530d62a4d9951e588c966c5371dc5e7918c25cc776fb24cc9bbe0dd->doDisplay(Array, Array)
#18 /vagrant/vendor/twig/twig/src/Template.php(380): Twig\Template->displayWithErrorHandling(Array, Array)
#19 /vagrant/vendor/twig/twig/src/Template.php(392): Twig\Template->display(Array)
#20 /vagrant/vendor/twig/twig/src/TemplateWrapper.php(45): Twig\Template->render(Array, Array)
#21 /vagrant/vendor/twig/twig/src/Environment.php(318): Twig\TemplateWrapper->render(Array)
#22 /vagrant/vendor/craftcms/cms/src/web/View.php(389): Twig\Environment->render('settings/genera...', Array)
#23 /vagrant/vendor/craftcms/cms/src/web/View.php(450): craft\web\View->renderTemplate('settings/genera...', Array)
#24 /vagrant/vendor/craftcms/cms/src/web/Controller.php(241): craft\web\View->renderPageTemplate('settings/genera...', Array, 'cp')
#25 /vagrant/vendor/craftcms/cms/src/controllers/SystemSettingsController.php(98): craft\web\Controller->renderTemplate('settings/genera...', Array)
#26 [internal function]: craft\controllers\SystemSettingsController->actionGeneralSettings()
#27 /vagrant/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#28 /vagrant/vendor/yiisoft/yii2/base/Controller.php(180): yii\base\InlineAction->runWithParams(Array)
#29 /vagrant/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction('general-setting...', Array)
#30 /vagrant/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('general-setting...', Array)
#31 /vagrant/vendor/craftcms/cms/src/web/Application.php(274): yii\base\Module->runAction('system-settings...', Array)
#32 /vagrant/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('system-settings...', Array)
#33 /vagrant/vendor/craftcms/cms/src/web/Application.php(259): yii\web\Application->handleRequest(Object(craft\web\Request))
#34 /vagrant/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#35 /vagrant/web/index.php(21): yii\base\Application->run()
#36 {main}
```

### Additional info

- Craft version: "3.5.17.1"
- PHP version: PHP 7.3.19-1~deb10u1
- Database driver & version:
- Plugins & versions:

    "barrelstrength/sprout-active": "^2.1",
    "born05/craft-assetusage": "^2.0",
    "codemonauts/craft-readonly-field": "^1.0",
    "craftcms/redactor": "2.8.5",
    "lukeyouell/craft-sentry": "^1.5",
    "nystudio107/craft-seomatic": "3.3.26",
    "solspace/craft-freeform": "3.9.9",
    "spicyweb/craft-neo": "2.8.15.1",
    "verbb/field-manager": "2.2.1",
    "verbb/navigation": "1.4.9",
    "verbb/super-table": "2.6.5",

**Comment by rauwebieten** [comment]
For some reason I was missing some keys in my project-config file. (name, timezone, live)
I have no idea why I am missing these.
By adding them manually this was fixed.

**Comment by brandonkelly** [comment]
Hm strange, but yeah that would definitely be a problem.

**Comment by matfish2** [comment]
I had the same issue, manually adding the keys to `project.yaml` didn't work for me. Instead I temporarily commented out the `value` key for `name`, live` and `timeZone` under `vendor/craftcms/cms/src/templates/setings/general/_index.html'. That would force the form to load. Then I saved the settings and reverted the changes.

**Comment by brandonkelly** [comment]
Just released Craft 3.7.13 with a fix for this, for anyone else that hits it.
